mleap-core/src/main/scala/ml/combust/mleap/core/classification/OneVsRestModel.scala

```diff
@@ -29,12 +29,17 @@ case class OneVsRestModel(classifiers: Array[ProbabilisticClassificationModel],
   def predictAll(features: Vector): (Double, Vector, Double) = {
     val predArray = Array.fill[Double](classifiers.length)(0.0)
     val (prediction, probability) = classifiers.zipWithIndex.map {
-      case (c, i) =>
+      case (c:ProbabilisticClassificationModel, i) =>
```

Comment:
We need changes in OVR as well, because if a non probabilistic models are built using OVR, then it should return the raw prediciton instead of probability as it is not available. So we need to have a case class which identifies which type of model is using OVR.